scripts/valhalla_build_config

```diff
@@ -499,7 +503,7 @@ if __name__ == '__main__':
     for k in keys[:-1]:
       v = v[k]
     v[keys[-1]] = getattr(args, leaf.replace('\0', '_'))
-    if v[keys[-1]] is None:
+    if isinstance(v[keys[-1]], type(optional())):
```

Comment:
here we remove config items that the user didnt set. before we used `None` to signify this, now we use `optional`